simulation/README.md

```diff
@@ -44,8 +42,8 @@ Make sure you have some version of python installed on your personal computer. *
 
 ## Running the Simulation
 ### Docker Simulation Setup
-1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2200 duke@[ip address]`).
-2. Run `source /opt/ros/kinetic/setup.bash` and `roscore &`.
+1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2201 duke@[ip address]`).
+2. Run `source /opt/ros/melodic/setup.bash` and `roscore &`.
```

Comment:
Tell them to use the build script, as stated in the main readme. Honestly, I think you can delete these two steps (1 and 2), since it is something that you do every single time to want to use the landside container. You could also just point to the main readme and say follow that. @rabinowitz what are your thoughts?

Reply:
This is my bad, I looked and looked for docker documentation in the various repos but failed to check the main readme. I think step 2 should go, but it's worth keeping step 1 and saying that they should be in the landside container.

I'll also make further changes to the readme when I do the code cleanup, which will almost certainly not be done tonight, but hopefully tomorrow.